### PR TITLE
Update User Profile

### PR DESCRIPTION
**Pull request by ekanshgupta123**
change profile component to show membership expiration dates to admin
Screen Shot 2022-07-06 at 12 54 28 PM

### CODE REVIEW

File: src/Pages/Profile/admin/editor-form.css [hunk 1]
```diff
@@ -5,3 +5,18 @@
...
+    border: 4px transparent rgb(134, 133, 133);
+    margin-top: 10px;
+    margin-left: 10px;
+    background: white;
+    justify-content: center;
+  }
+
+h5 {
+    color: 	#201c1c;
+    line-height: 10px;
```
Comment (neelp03) [inline src/Pages/Profile/admin/editor-form.css:20]: syntax got messed up here

File: test/frontend/Profile.test.js [hunk 2]
```diff
@@ -29,17 +29,6 @@ describe('<Profile />', () => {
 
   const wrapper = mount(<Profile user={user} />);
```
Comment (neelp03) [inline test/frontend/Profile.test.js:31]: Where are the tests for this component?

File: src/Pages/Profile/admin/Profile.js [hunk 3]
```diff
@@ -1,42 +1,36 @@
...
-          src='images/SCE-glow.png'
-        />
-      </Container>
-
-      <Badge color='primary'>
-        {enums.membershipStateToString(props.user.accessLevel)}
-      </Badge>
-      <h3>
+    <div className='wrapper'>
+      <h3 style = {{fontWeight: 'bold', textAlign: 'center'}}>
```
Comment (neelp03) [inline src/Pages/Profile/admin/Profile.js:8]: do styling in css file to keep it clean

**Review by neelp03 — CHANGES_REQUESTED** [review]
Checked it locally and everything works. Just a few minor suggestions below.

**Review by neelp03 — APPROVED** [review]

File: src/Pages/Profile/admin/editor-form.css [hunk 4]
```diff
@@ -5,3 +5,23 @@
...
+.wrapper {
+    width: 26rem;
+    height: fit-content;
+    border: 4px transparent rgb(134, 133, 133);
+    margin-top: 10px;
+    margin-left: 10px;
+    background: white;
+    justify-content: center;
+  }
+h3 {
```
Comment (akshtshrma24) [inline src/Pages/Profile/admin/editor-form.css:17]: scope these otherwise it will do global h3 css

**Review by neelp03 — APPROVED** [review]

**Review by evanugarte — APPROVED** [review]

**Pull request merged**